### DISCUSSION TITLE
Number of significant digits of battery level

### DISCUSSION THREAD

**Discussion by vwt12eh8** [vwt12eh8/hassio-ecoflow#38] [Polls]
![image](https://user-images.githubusercontent.com/63479906/184107023-19f56247-08fa-4aeb-84c8-1d4a1b86710f.png)

The battery level of the DELTA series can be obtained by a float.
But I think this number of digits is too much.

Please give me your opinion if I should limit the number of significant digits of this.

By the way, limiting the number of significant digits also affects the fineness of the graph.

The following is a comparison when the number of significant digits is set to 2
![image](https://user-images.githubusercontent.com/63479906/184109852-1ea8904d-1c9c-4cbd-8df4-38767d109f9f.png)

**Comment by skpow55** [comment]
I think limiting the decimal positions/float two 2 digits is ok.    
Beyond 2 digits is excessive for viewing battery level  in my opinion.

**Comment by mattwells** [comment]
I agree with 2 decimel places is more than sufficent, it is what you see in the Ecoflow app and is down to 0.36Wh on a Delta Pro, although smooth lines on the graph xD

**Comment by JasonMM2000** [comment]
Thank you for the people who are putting this all together!  I agree two decimal places would be fine, although I'd be good with just one decimal place.

**Comment by Ne0-Hack3r** [comment]
I created custom sensors to round to 1 decimal place (XX.X%) as well as custom sensors to combine/average Delta Pro Main + Extra Battery as well as two sets of DP+EB into a "total battery pack average" sensor...

I do think .1% is enough granularity personally but .01 is fine with me as well... any more digits makes display of values on dashboards more difficult.

Thank you again for all the fine work on this project!
